internal/publish/model/stats.go

```diff
@@ -0,0 +1,103 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package model is a model abstraction for health authority telemetry.
+package model
+
+import (
+	"time"
+)
+
+const (
+	MaxOldestTEK = 14
+	MaxOnsetDays = 28
+
+	PlatformAndroid = "android"
+	PlatformIOS     = "ios"
+	PlatformUnknown = "unknown"
+)
+
+var (
+	platforms = map[string]int{
```

Comment:
This feels like a potentially unnecessary optimization. I think we might be better off with a:

```go
func PlatformToInt(s string) int64 {
  switch s {
  case "android":
    return 1
  case "ios":
    return 2
  }

  return 0
}
```

I'm also very hesitant about having a "real" platform being mapped to the zero value.

Reply:
made it so unknown is 0